virt/virt-web-console-overview.adoc

```diff
@@ -879,6 +882,36 @@ Click the Options menu {kebab} beside a snapshot to select *Edit labels*, *Edit
 |====
 =====
 
+[id="ui-virtualmachine-details-diagnostics_{context}"]
+==== Diagnostics tab
+
+You can view the status conditions and volume snapshot status on the *Diagnostics* tab.
+
+.Diagnostics tab
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+|====
+|Element
+|Description
+
+|*Status conditions* table
+|List of conditions reported for all aspects of a VM
+
+|*Filter* field
+|Filter status conditions by category and condition.
+
+|*Search* field
+|Search status conditions by reason.
+
+|*Manage columns* icon 
+|Select columns to display.
+
+|*Volume snapshot* table
+|Snapshot status of VirtualMachine volumes.
```

Comment:
I agree with the suggestions you've provided (thanks!) except for this one, because the user is not actually taking a snapshot of the status/system. It shows the user if the VM is in a state that it can be "snapshotted" (for lack of a better word). Hopefully that makes sense, and if so, any recommendations on how to rewrite that?

Reply:
The snapshot status table looks like this:
![image](https://user-images.githubusercontent.com/31725355/234816451-06ce50b3-d82b-4819-86d7-e0248b8d9c63.png)

So the text would be something like this - "List of volumes, their snapshot enablement status, and reason"